.rubocop.yml

```diff
@@ -0,0 +1,8 @@
+AllCops:
```

Comment:
the updated file still doesn't pass the check on rubocop 1.50.2 (and there are a lot of notices spit out when run)

Reply:
Hi nanaya thanks for the feedback. I'm going to spend time and review the rubocop changes and your feedback below. If you have any other suggestions that I could make please let me know 🙂

Reply:
Sorry for the delay. I recently pushed new changes that applied your suggestions & feedback, while also fixing some assignment branch condition errors. This seems to pass on my end with rubocop 1.50.2.